packages/components/src/components/dropdown/dropdown.js

```diff
@@ -162,7 +162,7 @@ class Dropdown extends mixin(
         (listNode || this.element).focus();
         if (listNode) {
           const selectedNode = listNode.querySelector(
-            this.options.selectorItemSelected
+            `${this.options.selectorItemSelected} ${this.options.selectorItem}`
```

Comment:
We don't do string interpolation for selectors in code. In this kind of case, we introduce a new option, like `selectorLinkSelected`.